test/gateway/ost_prime/constructor.js

```diff
@@ -0,0 +1,81 @@
+const OSTPrime = artifacts.require("OSTPrime")
+  , BN = require('bn.js')
+  , web3 = require('../../test_lib/web3.js');
+
+const Utils = require('../../../test/test_lib/utils');
+
+const NullAddress = "0x0000000000000000000000000000000000000000";
```

Comment:
utils.js has "NullAddress" variable which can be used instead of defining a new variable here.

Reply:
`NullAddress` is not exposed. If we all agree we can create a new ticket for this.